frontend/src/service/Faq.jsx

```diff
@@ -326,3 +326,26 @@ export const downvote = async (questionId, handleToast) => {
     throw new Error("Failed to downvote question");
   }
 };
+
+export const postAnswer = async (data, setToast) => {
+  try {
+    showToast(setToast,"Posting...","info")
+    const url = `${END_POINT}/answers/`;
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "content-type": "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+    const res = await response.json();
+    if(response.status==200)
+      showToast(setToast, "Answer posted successfully","success");
```

Comment:
Can you show the message like “Thanks for answering, it has been sent to admins for review and will appear here on approval “

Reply:
Okay i will change it....But Build not shown any error in local machine.. it failing here